### ISSUE TITLE
[a11y]SearchBar missing searchbox role

### ISSUE THREAD

**Issue by chunhtai** [flutter/flutter#161540]
When using this widget https://api.flutter.dev/flutter/material/SearchBar-class.html

| OS | role |
|--------|--------|
| web | searchbox |
| ios | [UIAccessibilityTraitSearchField](https://developer.apple.com/documentation/uikit/uiaccessibilitytraits/searchfield?language=objc) |
| macos |  NSAccessibilitySearchFieldSubrole |
| windows | ROLE_SYSTEM_TEXT |
| android | SearchView |
